packages/contracts/docs/osx/01-how-it-works/02-framework/02-plugin-management/02-plugin-setup/index.md

```diff
@@ -1,72 +1,78 @@
 ---
-title: Setup
+title: The Plugin Contracts
 ---
 
-## The Plugin Setup Process
+## The Smart Contracts Behind Plugins
 
-A DAO can be set up and customized by the **installation**, **update, and** **uninstallation** of plugins.
-In this section you will learn how the plugin setup process and related infrastructure in Aragon OSx works.
+A DAO can be set up and customized by the **installation, update, and uninstallation** of plugins. Plugins are composed of two key contracts:
 
-In order for a plugin to function, associated contracts need to be deployed and gathered, often requiring permissions from the DAO.  
-For example, a governance plugin will need permission to call the `execute` function in the DAO.
+- **Plugin contract:** contains the plugin's implementation logic; everything needed to extend the functionality for DAOs.
+- **Plugin Setup contract:** contains the instructions needed to install, update, and uninstall the plugin into the DAO. This is done through granting or revoking permissions, enabling the DAO to make use of the plugin's functionality.
 
-The required setup logic is written and taken care off by the plugin developer in the `PluginSetup` contract they create and that is associated with each `Plugin` contract version release (see [Developing a Plugin](../index.md)). The `PluginSetup` contract then interacts with the Aragon OSx framework so that installing, updating, and uninstalling a plugin to a DAO through the UI becomes very simple for the DAO end-user.
+![Aragon OSx Plugins](https://res.cloudinary.com/dacofvu8m/image/upload/v1683225098/Screen_Shot_2023-05-04_at_14.31.25_r0qqut.png)
 
-Except for the gas costs required, the installation, update, and uninstallation of plugins is completely free.
+How this works:
 
-### Security Considerations
+- Although a Plugin is composed `Plugin` and `PluginSetup` contracts, the Aragon OSx protocol only knows of the `PluginSetup` contract.
+- Since the `PluginSetup` contract is the one in charge of installing the plugin into a DAO, it is the one also in charge of deploying the Plugin instace specific to that DAO with the parameters set by the DAO in advance. You can review how to build a `PluginSetup` contract [here](../../../../02-how-to-guides/02-plugin-development/index.md).
```

Comment:
`in charge of installing the plugin into a DAO` => This is not done by the plugin setup, but by someone with permissions (PSP).

The PluginSetup does the deployments and configuration but a plugin is considered to be installed when the permissions it needs are granted

Reply:
Much better wording. Thanks! Edited.